renderdoc/core/replay_proxy.cpp

```diff
@@ -1707,6 +1707,21 @@ bool ReplayProxy::SendReplayCommand(ReplayProxyPacket type)
   return true;
 }
 
+// Modify the params that are used when getting the texture data for replay on the current driver.
+void ReplayProxy::ConvertProxyTextureIfNeeded(FetchTexture &tex, GetTextureDataParams &params)
+{
+  if(m_Proxy->IsTextureFormatSupported(tex.format))
+    return;
+
+  tex.format.special = false;
+  tex.format.compCount = 4;
+  tex.format.compByteWidth = 1;
+  tex.format.compType = eCompType_UNorm;
+
+  params.forceRGBA8unorm = true;
+  params.whitePoint = 1.0f;
```

Comment:
Yeah it makes sense to have the remap type choice in common code. Im making the choice based on the compByteWidth, as remapping is about not losing precision as you said.

As the range adaptation is only for RGBA8, I put the whitepoint line into that remap case label.